posthog/api/cohort.py

```diff
@@ -183,6 +186,13 @@ def update(self, cohort: Cohort, validated_data: Dict, *args: Any, **kwargs: Any
 
         return cohort
 
+    def to_representation(self, instance):
```

Comment:
I believe this is what you were looking for @neilkakkar. As mentioned in the other comment I closed, using a serializermethodfield will override the django parsing magic. Quite annoying how much magic is wrapped in django T_T

Reply:
ohh T.T, nice catch!